configs/windows/MinimalAgent/nanoserver/NanoServer2022.Dockerfile

```diff
@@ -28,12 +28,14 @@ USER ContainerAdministrator
 COPY scripts/*.cs /scripts/
 SHELL ["pwsh", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
-# CaseSensitivity is essential for the agent => enforcing it. SHELL usage is mandatory (pwsh or other won't work)
-RUN mkdir C:\\BuildAgent
-SHELL ["pwsh", "-Command", "fsutil.exe", "file", "SetCaseSensitiveInfo", "C:\\BuildAgent", "enable"]
```

Comment:
This line did not have any effect, as `fsutil` is not available within neither of the base images. The error was silently ignored by the build process due to `SilentlyContinue` flag specified by the line above.